acms-react-app/src/Pages/Admin_Pages/Invoices/Invoices_Page/AddNewInvoice.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Navbar from '../../../../Components/Navigation_Bars/Logged_In/NavBar.js';
+import Footer from '../../../../Components/Footer/Footer.js';
+import adminService from '../../../../Services/admin.service.js';
+
+function AddNewInvoice() {
+    const navigate = useNavigate();
```

Comment:
For this page you forgot to add the authorization. Take examples from other pages where I used userService to authorize and display a message if the user is unauthorized.

Reply:
My fault I forgot about auth in front-end